VKApp/VKApp/Views/GroupUserTableViewCell.swift

```diff
@@ -19,10 +19,10 @@ final class GroupUserTableViewCell: UITableViewCell {
 
     // MARK: - Public Methods
 
-    func configure(group: VKGroups, vkNetworkService: VKNetworkService) {
+    func configure(group: VKGroups, image: UIImage?) {
```

Comment:
```suggestion
    func configure(group: VKGroups, image: UIImage?) {
```
сервис вместо картинки передай